apm-agent-core/src/test/java/co/elastic/apm/agent/impl/circuitbreaker/CircuitBreakerTest.java

```diff
@@ -274,7 +281,7 @@ private static void awaitAssert(ThrowingRunnable assertion) {
     private static ConditionFactory doAwait() {
         return await()
             .pollInterval(1, TimeUnit.MILLISECONDS)
-            .timeout(50, TimeUnit.MILLISECONDS);
+            .timeout(200, TimeUnit.MILLISECONDS);
```

Comment:
also, it seems that some failures were due to timeout being too short, a few measurements showed that it sometimes took between 30ms, thus there wasn't a sufficient margin under system load.